src/main.dev.js

```diff
@@ -336,6 +344,83 @@ if (isLinux) {
   });
 }
 
+if (isWindows && isWindows10) {
```

Comment:
You can probably remove the `&& isWindows10`.

If the isWindows10 test is working on your Win11 machine, then I probably need to fix that validation since it's only supposed to target Win10.

Reply:
```
> os.release()
'10.0.22000'
```

This is what Windows 11 shows, that's why the regex also works there.